src/pages/starter-kit/checkout/shipping-reference.md

```diff
@@ -19,19 +19,38 @@ To configure out-of-process shipping methods, add shipping carriers using the RE
 
 The raw Shipping REST schema is available [here](/shipping.xml).
 
-| **Route URL**                     | **Method** | **Description**                                |
-|-----------------------------------|------------|------------------------------------------------|
-| `/V1/oope_shipping_carrier`       | GET        | Retrieve a list of all out-of-process shipping carriers  |
+| **Route URL**                     | **Method** | **Description**                                          |
+| --------------------------------- | ---------- | -------------------------------------------------------- |
+| `/V1/oope_shipping_carrier`       | POST       | Create a new shipping carrier.                           |
+| `/V1/oope_shipping_carrier`       | PUT        | Update a shipping carrier.                               |
 | `/V1/oope_shipping_carrier/:code` | GET        | Retrieve an out-of-process shipping carrier by its code. |
-| `/V1/oope_shipping_carrier`       | POST       | Create a new shipping carrier.                 |
-| `/V1/oope_shipping_carrier`       | PUT        | Update a shipping carrier.                     |
-| `/V1/oope_shipping_carrier/:code`   | DELETE     | Delete shipping carrier by code                |
+| `/V1/oope_shipping_carrier`       | GET        | Retrieve a list of all out-of-process shipping carriers  |
+| `/V1/oope_shipping_carrier/:code` | DELETE     | Delete shipping carrier by code                          |
+
+### Create a new OOPE shipping carrier
+
+The POST `/V1/oope_shipping_carrier` creates an out-of-process shipping carrier in the Adobe Commerce instance.
```

Comment:
```suggestion
The POST `/V1/oope_shipping_carrier` endpoint creates an out-of-process shipping carrier in the Adobe Commerce instance.
```

Review all entries with this format so that we either remote "The" or add a word like "endpoint" after the path.